### ISSUE TITLE
`rem()` is not available in React Server Component

### ISSUE THREAD

**Issue by SukkaW** [mantinedev/mantine#5119]
### What package has an issue

@mantine/core

### Describe the bug

The `rem()` function is not available in the React Server Component. When importing `rem` from a React Server Component, the following error appears:

```
Error: Attempted to call rem() from the server but rem is on the client. It's not possible to invoke a client function from the server, it can only be rendered as a Component or passed to props of a Client Component.
```

### What version of @mantine/* packages do you have in package.json? (Note that all @mantine/* packages must have the same version in order to work correctly)

7.1.5

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

### Possible fix

It appears that the published `rem.mjs` and `rem.js` contain the `'use client'` directive, which is completely unnecessary.

**Comment by SukkaW** [comment]
Current workaround: use https://foxact.skk.moe/rem

```diff
- import { rem } from '@mantine/core' 
+ import { mantine_rem as rem } from 'foxact/rem';
```

**Comment by rtivital** [comment]
Fixed in [7.2.1](https://github.com/mantinedev/mantine/releases/tag/7.2.1).
